model-optimizer/mo/main.py

```diff
@@ -429,4 +457,6 @@ def main(cli_parser: argparse.ArgumentParser, framework: str):
 
 if __name__ == "__main__":
     from mo.utils.cli_parser import get_all_cli_parser
-    sys.exit(main(get_all_cli_parser(), None))
+    from mo.front_ng.frontendmanager_wrapper import create_fem
```

Comment:
we tend to not use local imports where possible. Is it really mandatory to import module here or it can be moved to the top of the file?